src/main/python/image_preprocessing/template_matching.py

```diff
@@ -44,9 +45,28 @@ def identify(self, source):
             # find the template in the source image
             result = cv2.matchTemplate(resized, template_image, cv2.TM_CCOEFF_NORMED)
 
-            (_, maximum_value, minLoc, (x, y)) = cv2.minMaxLoc(result)
+            (_, maximum_value, _, _) = cv2.minMaxLoc(result)
 
             if (maximum_value > threshold):
                 print(object_identifier)
                 return object_identifier
+
+        for scale in np.linspace(0.8, 1.8, 20)[::-1]:
```

Comment:
Please document these magic numbers, i.e. add a comment above this line that explains each one.